src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -100,6 +103,9 @@ public static int MainInternal(string[] args, CommandOutputLogger log, IEnvironm
 
                     PackageSearchCommand.Register(packageCommand, getHidePrefixLogger);
                     PackageUpdateCommand.Register(packageCommand, interactiveOption);
+#if DEBUG
+                    PackageDownloadCommand.Register(packageCommand, interactiveOption);
```

Comment:
why is this being added only for debug?  Other commands don't do this.

Reply:
To Andy's point, we should follow the existing convention/pattern if possible or change all the commands to your proposed pattern.